router/comment.js

```diff
@@ -0,0 +1,124 @@
+import express from "express";
+import { PrismaClient } from "@prisma/client";
+import { validateContent, validateId } from "../middleware/validate.js";
+import { BadRequestError, InternalServerError } from "../middleware/error.js";
+
+const prisma = new PrismaClient();
+const router = express.Router();
+
+// 마켓 댓글 + 목록 조회
+router
+  .route("/product/:id")
+  .post(validateContent, validateId, async (req, res) => {
+    try {
+      const productId = parseInt(req.params.id);
+      const { content } = req.body;
+
+      const comment = await prisma.comment.create({
+        data: {
+          content,
+          product: { connect: { id: productId } },
+        },
+      });
+      res.status(201).json(comment);
+    } catch (e) {
+      next(new InternalServerError("댓글 작성에 실패했습니다."));
+    }
+  })
+  .get(validateId, async (req, res) => {
+    try {
+      const productId = parseInt(req.params.id);
+      const take = parseInt(req.query.take) || 10;
+      const cursor = req.query.cursor ? parseInt(req.query.cursor) : null;
+
+      const comment = await prisma.comment.findMany({
+        where: { productId },
+        take: take,
```

Comment:
가능하면 아래 코드와 일관성을 맞추기 위해 shorthand property를 사용해주세요! 
```suggestion
        take,
```